ticketing.md

```diff
@@ -0,0 +1,28 @@
+# Ticketing
+
+This team is responsible for making sure that PyCon's ticket system is working and that the money is properly accounted for. Because this team involves money, the treasurer of Python.Ph will be the de-facto leader of this committee.
+
+There should be a cut-off for tickets sold online. This is to give the entire PyCon team some breathing room before PyCon. It will be under the discretion of the PyCon head if tickets will still be sold at the event.
+
+During the conference, merchandise will be sold. As a general rule, no change will be given on anything sold at PyCon. The change will instead be considered as donation to Python.Ph. The exact nature of the donation will be determined beforehand, but on previous PyCons, it was decided that the donation will be used to fund local Python groups.
```

Comment:
This practice was announce on the last PyCon regarding the tshirts and when selling the tickets. We received a lot of support with one participant paying twice.

I'll put up an issue ticket on this so we can discuss in more depth. Maybe its the way it was written that made it sounds fishy (in my defense, I was under the influence of coffee :coffee:).